CleverPush/CleverPush/Source/CPNotification.h

```diff
@@ -19,6 +19,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readwrite) BOOL silent;
 @property (nonatomic, strong, nullable) NSDate *createdAt;
 @property (nonatomic, strong, nullable) NSDate *expiresAt;
+@property (nonatomic) BOOL multiscreenEnable;
```

Comment:
I think this is not needed?

Reply:
when we tap on badge and then it will represent on the screen At that time for exception I have keep this code.